apple/RNCWebView.m

```diff
@@ -37,7 +37,7 @@ -(id)inputAccessoryView
   if (_webView == nil) {
     return nil;
   }
-  
```

Comment:
shouldn't the PR  don't touch extra whitespace to ease the size of the diff and increase chance of it being merged ?

Reply:
@matiaskorhonen  ping ? I think removing the extra whitespace change will increase a lot the chance of it being reviewed/merged